### PR TITLE
Feature/no project page

### PR DESCRIPTION
**Pull request by ChrisBerg**
Added popups to the home page, from which new projects can be created and existing projects can be loaded as current project.

### CODE REVIEW

File: src/main/resources/.h2.server.properties [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #H2 Server Properties
-#Fri May 20 20:28:19 CEST 2016
-0=H2 (Disk)|org.h2.Driver|jdbc\:h2\:file\:./target/h2db/db/thefirstorder|thefirstorder
-1=H2 (Memory)|org.h2.Driver|jdbc\:h2\:mem\:thefirstorder|thefirstorder
+#Thu May 26 17:45:47 CEST 2016
+0=H2 (Memory)|org.h2.Driver|jdbc\:h2\:mem\:thefirstorder|thefirstorder
```
Comment (hungnguyen94) [inline src/main/resources/.h2.server.properties]: No need to commit this?

File: src/main/webapp/app/home/create-dialog.controller.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
+(function () {
```
Comment (hungnguyen94) [inline src/main/webapp/app/home/create-dialog.controller.js]: Is this for the Project manager dialog? If so, I would change the name. From just the name `create-dialog.controller.js`, it isn't very clear what it's purpose is.

File: src/main/webapp/app/home/load-dialog.controller.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        .controller('LoadDialogController', LoadDialogController);
+
+    LoadDialogController.$inject = ['$timeout', '$scope', '$state', '$stateParams', '$uibModalInstance', '$q', 'entity', 'Project', 'ProjectManager'];
+
+    function LoadDialogController($timeout, $scope, $state, $stateParams, $uibModalInstance, $q, entity, Project, ProjectManager) {
+        var vm = this;
+        vm.project = entity;
+
+        vm.loadAllProjects = loadAllProjects;
+        vm.loadAllProjects();
```
Comment (hungnguyen94) [inline src/main/webapp/app/home/load-dialog.controller.js]: You can replace this with 
``
vm.projects = Project.query();
``

**Comment by hungnguyen94** [comment]
I added a few notes.   
Don't commit `src/main/webapp/index.html`. 
Other than that, it looks good.

**Comment by cadw** [comment]
Looks fine to me, but as Hung said, committing the index.html is not a great idea. For some reason, index.html is overwritten as soon as you pull, maybe we should create an issue for that.

**Comment by ChrisBerg** [comment]
Thanks for the remarks, i modified the code

**Comment by cadw** [comment]
All checks have passed and you modified the code with our remarks, I am merging it right now. Good job!

**Pull request merged**